iron-overlay-manager.html

```diff
@@ -78,10 +78,58 @@
     },
 
     /**
+     * Brings the overlay at the specified index to the front.
+     * @param {number|Element} i
+     */
+    bringToFront: function(i) {
+      // If not a number, assume i is the overlay.
+      if (typeof i !== 'number') {
+        i = this._overlays.indexOf(i);
+      }
+      var overlay = this._overlays[i];
+      var lastI = this._overlays.length - 1;
+      // If already the top element, return.
+      if (!overlay || i === lastI) {
+        return;
+      }
+      // Update z-index to be on top.
+      var minimumZ = Math.max(this.currentOverlayZ(), this._minimumZ);
```

Comment:
Ah, nevermind. I thought `_minimumZ` was supposed to update to reflect the highest `zIndex` used.